docs/deployment-artifacts.md

```diff
@@ -320,7 +320,10 @@ type ExecutionArtifact = {
     isModuleDeployed: boolean
     isExecuting: boolean
   }
-  unlabeledAddresses: Array<string>
+  unlabeledContracts: Array<{
+    address: string
+    initCodeWithArgs: string
+  }>
```

Comment:
The `ExecutionArtifact` now includes each unlabeled contract's `initCodeWithArgs` in addition to its address. This'll make it easier to verify unlabeled contracts after a deployment is completed. This change isn't related to the main focus of the PR, but it was natural to add it because the `ParsedConfig`, which is one of the main inputs to the `ExecutionArtifact`, now contains the same `unlabeledContracts` array. (The `ParsedConfig` needs the `initCodeWithArgs` so that we can use it to filter out the `CREATE3` proxy initcode in the preview).